### ISSUE TITLE
Identifying and allocating to a GPU

### ISSUE THREAD

**Issue by magick93** [sub-mod/tensorflow-serving-gpu-s2i#1]
Hello

When creating a GPU based openshift application, Im asked for the `GPU_NAME` and also `GPU_LIMITS`.

How are these values used?

Im specifically interested in how I allocate a project to run on a specific GPU, or collection of GPUs. Is it possible to add the identity of the GPUs to the project?

**Comment by sub-mod** [comment]
I don't think you can limit GPU to a namespace.But i need to double check that.
Here are my notes to setup [Openshift with GPU](https://gist.github.com/sub-mod/68bf5483241bd118aa09c27a8289db06) 

if you have a GPU_NAME you can select the correct GPU as below
```
"matchExpressions": 
[
    {
     "key": "alpha.kubernetes.io/nvidia-gpu-name",
     "operator": "In",
      "values": [  "GPU_NAME"  ]
      }
]
```

With GPU_LIMITS you can control how many GPUs are needed for the deployment config i.e the application
```
"resources": {
  "limits": {
    "alpha.kubernetes.io/nvidia-gpu": "1"
   }
},
```

**Comment by magick93** [comment]
Hi @sub-mod 

I maybe mistaken, but I thought nvidia identifies the GPUs by index number, not name.

Otherwise, what should the name be? How do you name a GPU?
